source/jormungandr/jormungandr/parking_space_availability/bss/tests/atos_test.py

```diff
@@ -81,6 +86,12 @@ def parking_space_availability_atos_get_informations_test():
     provider = AtosProvider(u'10', u'vélitul', u'https://webservice.atos.com?wsdl', {'keolis'})
     provider.get_all_stands = MagicMock(return_value=all_stands)
     assert provider.get_informations(poi) == stands
+    invalid_poi = {}
+    assert provider.get_informations(invalid_poi) is None
+
+    poi_blur_ref = {'properties': {'ref': '02'}}
+    assert provider.get_informations(poi_blur_ref) == stands
```

Comment:
The ref of the poi returned by the connector is `2`, this test check that we ignore leading 0